src/Language/Haskell/TH/Lens.hs

```diff
@@ -1738,52 +1668,29 @@ _Interruptible
 -- |
 -- @
 -- _InlineP :: 'Prism'' 'Pragma' ('Name', 'Inline', 'RuleMatch', 'Phases') -- template-haskell-2.8+
```

Comment:
Now that the API for `_InlineP` is uniform across all supported versions of `template-haskell`, this Haddock comment is completely redundant. I'd favor removing it entirely.